test/runtests.jl

```diff
@@ -257,14 +257,37 @@ end
         @test Matrix(HA.Q) ≈ Matrix(HB.Q)
     end
 end
+@testset "pfaffian.jl" begin
+    for n in [2,4,6]
+        A=rand(1:10,n,n)
+        for i=1:n
+            A[i,i]=0
+            for j=i+1:n
+                A[i,j]=-A[j,i]
+            end
+        end
+        pf = SLA.exactpfaffian!(copy(A))
+        @test pf*pf ≈ det(A)
+        A=SLA.skewhermitian(randn(n,n))
+        pf= SLA.realpfaffian(A)
+        @test pf*pf ≈ det(A.data)
```

Comment:
Now you can check
```
@test pfaffian(A) ≈ pfaffian(BigInt.(A)) == pfaffian(SkewHermitian(BigInt.(A)))
```
and this will call the `pfaffian!` methods etcetera.

Reply:
I would also try a couple of random integer matrices with another package for the Pfaffian, and check that you can reproduce those (just hardcode the matrices and the results into the test) to make sure that you get the right sign.